### PR TITLE
Add fill_int in Friesian fillna

### PR DESCRIPTION
**Pull request by cyita**

### CODE REVIEW

File: pyzoo/zoo/friesian/feature/table.py [hunk 1]
```diff
@@ -100,20 +97,31 @@ def drop(self, *cols):
         """
         return self._clone(self.df.drop(*cols))
 
-    def fillna(self, value, columns):
+    def fillna(self, value, columns, fill_int=False):
```
Comment (jason-dai) [inline pyzoo/zoo/friesian/feature/table.py:100]: why do we need the user to pass an extra parameter? can we infer that automatically?

Comment (cyita) [reply pyzoo/zoo/friesian/feature/table.py:100]: The current implementation will raise exception when columns contain a long/double type column. We can pass value like 2.0 to fill double columns but for long columns, we cannot fill with -2147483648 <= value <= 2147483647.

Comment (cyita) [reply pyzoo/zoo/friesian/feature/table.py:100]: Or we can check the type of columns here and if columns are all int type then we call fill_na_int

Comment (jason-dai) [reply pyzoo/zoo/friesian/feature/table.py:100]: Yes, it's better to check the the type of columns on the Python side.

**Comment by cyita** [comment]
Jenkins passed http://10.239.47.210:18888/job/ZOO-PR-Validation/5514/  http://10.239.47.210:18888/job/ZOO-PR-Validation/5505/

**Pull request merged**